### PR TITLE
Quote keyword column names in normalized destination output

### LINKED ISSUE #1561: Normalizing data from stripe fails
## Current Behavior
Stripe normalization (to redshift) is failing with the following exception 

```
020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 2 of 16 START table model stripe.charges..................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 1 of 16 START table model stripe.balance_transactions........................................................ [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 4 of 16 START table model stripe.customers................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 5 of 16 START table model stripe.disputes.................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 6 of 16 START table model stripe.events...................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 7 of 16 START table model stripe.invoice_items............................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 8 of 16 START table model stripe.invoice_line_items.......................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 9 of 16 START table model stripe.invoices.................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 10 of 16 START table model stripe.payout_transactions........................................................ [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 11 of 16 START table model stripe.payouts.................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 12 of 16 START table model stripe.plans...................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 13 of 16 START table model stripe.products................................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 14 of 16 START table model stripe.subscription_items......................................................... [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 15 of 16 START table model stripe.subscriptions.............................................................. [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 16 of 16 START table model stripe.transfers.................................................................. [RUN]
2020-12-30 23:03:32 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:32 | 3 of 16 START table model stripe.coupons..................................................................... [RUN]
2020-12-30 23:03:33 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:33 | 5 of 16 ERROR creating table model stripe.disputes........................................................... [[31mERROR[0m in 1.11s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 4 of 16 ERROR creating table model stripe.customers.......................................................... [[31mERROR[0m in 1.70s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 2 of 16 ERROR creating table model stripe.charges............................................................ [[31mERROR[0m in 1.76s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 8 of 16 ERROR creating table model stripe.invoice_line_items................................................. [[31mERROR[0m in 1.75s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 7 of 16 ERROR creating table model stripe.invoice_items...................................................... [[31mERROR[0m in 1.78s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 14 of 16 ERROR creating table model stripe.subscription_items................................................ [[31mERROR[0m in 1.80s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 13 of 16 ERROR creating table model stripe.products.......................................................... [[31mERROR[0m in 1.83s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 15 of 16 ERROR creating table model stripe.subscriptions..................................................... [[31mERROR[0m in 1.81s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 9 of 16 ERROR creating table model stripe.invoices........................................................... [[31mERROR[0m in 1.91s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 6 of 16 ERROR creating table model stripe.events............................................................. [[31mERROR[0m in 1.92s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 3 of 16 ERROR creating table model stripe.coupons............................................................ [[31mERROR[0m in 1.86s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 16 of 16 ERROR creating table model stripe.transfers......................................................... [[31mERROR[0m in 1.91s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 12 of 16 ERROR creating table model stripe.plans............................................................. [[31mERROR[0m in 1.98s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 11 of 16 ERROR creating table model stripe.payouts........................................................... [[31mERROR[0m in 2.06s]
2020-12-30 23:03:34 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:34 | 1 of 16 OK created table model stripe.balance_transactions................................................... [[32mSELECT[0m in 2.22s]
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:35 | 10 of 16 OK created table model stripe.payout_transactions................................................... [[32mSELECT[0m in 2.27s]
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:35 | 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 23:03:35 | Finished running 16 table models in 2.64s.
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [31mCompleted with 14 errors and 0 warnings:[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model disputes (models/generated/disputes.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/disputes.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model customers (models/generated/customers.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/customers.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model charges (models/generated/charges.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   syntax error at or near "order"
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   LINE 61: ...ast(paid as varchar), '') || '-' || coalesce(cast(order as v...
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -                                                                 ^
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/charges.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model invoice_line_items (models/generated/invoice_line_items.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/invoice_line_items.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model invoice_items (models/generated/invoice_items.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/invoice_items.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model subscription_items (models/generated/subscription_items.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/subscription_items.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model products (models/generated/products.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/products.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model subscriptions (models/generated/subscriptions.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/subscriptions.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model invoices (models/generated/invoices.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/invoices.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model events (models/generated/events.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/events.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model coupons (models/generated/coupons.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/coupons.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model transfers (models/generated/transfers.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/transfers.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model plans (models/generated/plans.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/plans.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - [33mDatabase Error in model payouts (models/generated/payouts.sql)[0m
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   cannot cast type text to boolean
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 -   compiled SQL at ../build/run/airbyte_utils/models/generated/payouts.sql
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - 
2020-12-30 23:03:35 INFO (/tmp/workspace/10/4) LineGobbler(voidCall):69 - Done. PASS=2 WARN=0 ERROR=14 SKIP=0 TOTAL=16
2020-12-30 23:03:35 DEBUG (/tmp/workspace/10/4) DefaultNormalizationRunner(close):97 - Closing tap process
2020-12-30 23:03:35 ERROR (/tmp/workspace/10/4) DefaultSyncWorker(run):117 - Normalization Failed.
io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultSyncWorker.run(DefaultSyncWorker.java:114) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
	at io.airbyte.workers.DefaultSyncWorker.run(DefaultSyncWorker.java:48) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
	at io.airbyte.workers.wrappers.OutputConvertingWorker.run(OutputConvertingWorker.java:44) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
	at io.airbyte.workers.wrappers.JobOutputSyncWorker.run(JobOutputSyncWorker.java:32) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
	at io.airbyte.scheduler.WorkerRun.lambda$new$0(WorkerRun.java:53) [io.airbyte-airbyte-scheduler-0.9.2-alpha.jar:?]
	at io.airbyte.scheduler.WorkerRun.call(WorkerRun.java:61) [io.airbyte-airbyte-scheduler-0.9.2-alpha.jar:?]
	at io.airbyte.scheduler.WorkerRun.call(WorkerRun.java:42) [io.airbyte-airbyte-scheduler-0.9.2-alpha.jar:?]
	at io.airbyte.commons.concurrency.LifecycledCallable.execute(LifecycledCallable.java:114) [io.airbyte-airbyte-commons-0.9.2-alpha.jar:?]
	at io.airbyte.commons.concurrency.LifecycledCallable.call(LifecycledCallable.java:98) [io.airbyte-airbyte-commons-0.9.2-alpha.jar:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) [?:?]
	at java.lang.Thread.run(Thread.java:832) [?:?]
	Suppressed: io.airbyte.workers.WorkerException: Tap process wasn't successful
		at io.airbyte.workers.normalization.DefaultNormalizationRunner.close(DefaultNormalizationRunner.java:100) ~[io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
		at io.airbyte.workers.DefaultSyncWorker.run(DefaultSyncWorker.java:109) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
		at io.airbyte.workers.DefaultSyncWorker.run(DefaultSyncWorker.java:48) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
		at io.airbyte.workers.wrappers.OutputConvertingWorker.run(OutputConvertingWorker.java:44) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
		at io.airbyte.workers.wrappers.JobOutputSyncWorker.run(JobOutputSyncWorker.java:32) [io.airbyte-airbyte-workers-0.9.2-alpha.jar:?]
		at io.airbyte.scheduler.WorkerRun.lambda$new$0(WorkerRun.java:53) [io.airbyte-airbyte-scheduler-0.9.2-alpha.jar:?]
		at io.airbyte.scheduler.WorkerRun.call(WorkerRun.java:61) [io.airbyte-airbyte-scheduler-0.9.2-alpha.jar:?]
		at io.airbyte.scheduler.WorkerRun.call(WorkerRun.java:42) [io.airbyte-airbyte-scheduler-0.9.2-alpha.jar:?]
		at io.airbyte.commons.concurrency.LifecycledCallable.execute(LifecycledCallable.java:114) [io.airbyte-airbyte-commons-0.9.2-alpha.jar:?]
		at io.airbyte.commons.concurrency.LifecycledCallable.call(LifecycledCallable.java:98) [io.airbyte-airbyte-commons-0.9.2-alpha.jar:?]
		at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
		at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) [?:?]
		at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) [?:?]
		at java.lang.Thread.run(Thread.java:832) [?:?]
```

## Steps to Reproduce
TODO

## Severity of the bug for you
High -- Stripe will not complete a sync without this

### PR DESCRIPTION
**Pull request by sherifnada**
## What
Closes #1561 

## How
Quotes reserved keywords if they are used as column names

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

### CODE REVIEW

**Review by ChristopheDuong — APPROVED** [review]
Let's merge my PR (https://github.com/airbytehq/airbyte/pull/1761) with an extra test cases into yours, then we're good to merge to master?

File: airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java [hunk 1]
```diff
@@ -255,7 +255,7 @@ public void testCheckConnectionInvalidCredentials() throws Exception {
     @Override
     public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
       return Stream.of(
-          Arguments.of("exchange_rate_messages.txt", "exchange_rate_catalog.json"),
+          // Arguments.of("exchange_rate_messages.txt", "exchange_rate_catalog.json"),
```
Comment (sherifnada) [inline airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java:258]:
```suggestion
          Arguments.of("exchange_rate_messages.txt", "exchange_rate_catalog.json"),
```

**Pull request merged**